.babelrc

```diff
@@ -7,7 +7,8 @@
           "svgo": false
         }],
         ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
-        "istanbul"
+        "istanbul",
```

Comment:
i think the istanbul transform needs to come last, so it can properly instrument the code for coverage